### ISSUE TITLE
[Request] Clarify type and limit of various fields

### ISSUE THREAD

**Issue by Irohaede** [sspanel-uim/WebAPI-documents#1]
WebAPI is based on JSON which supports many types (object, array, number, string, bool, or null)([RFC 4627 Section 2.1](https://datatracker.ietf.org/doc/html/rfc4627#section-2.1)).

But in some server implementations, the server would return wired data type(ex. null/string on a number field). It would make some unexpected problems or make server hard to parse each field.

Also, I request for size limit for number type fields(128, 64, 32, 16, 8 bit? unsigned?) which could improve node memory usage.

**Comment by M1Screw** [comment]
You are absolutely right about this one, the document for SSPanel is a total mess right now, hence I propose following changes to resolve the issue you mentioned and clarify the development process:

1. Merge sspanel-uim/API-documents and sspanel-uim/WebAPI-documents into a single "docs for dev" repo and update it according to the current development of panel.
2. Re-format each document part to a single human-readable .md file instead of vuepress.
3. Add an in-depth custom-config usage explanation to the development document.

**Comment by Irohaede** [comment]
I see... It seems to be hard work to reach that.
